core/templates/components/ck-editor-helpers/ck-editor-4-rte.component.html

```diff
@@ -1,6 +1,6 @@
 <div>
   <div class="d-flex flex-column-reverse">
-    <span class="cke-offline-warning">* Tools with dark background can not be used when offline.</span>
+    <span class="cke-offline-warning" *ngIf="!connectedToInternet">* Tools with dark background can not be used when offline.</span>
```

Comment:
```suggestion
    <span class="oppia-cke-offline-warning" *ngIf="!connectedToInternet">* Tools with dark background can not be used when offline.</span>
```

Specifying that this is oppia-specific class*

Reply:
Done.